docs/how-to/extending/unbound_expression.qmd

```diff
@@ -0,0 +1,102 @@
+# Write and execute unbound expressions
```

Comment:
```suggestion
---
title: Write and execute unbound expressions
---
```

ugh, sorry I missed this before -- the title of the browser tab is currently "unbound_expression" because it doesn't pick it up automatically from a top-level markdown header.  I think if you add this it will set the page header correctly and also give the tab a better name.